core/src/test/java/apoc/ArgumentDescriptionsTest.java

```diff
@@ -202,9 +200,11 @@ public void teardown() {
 
     @Test
     public void functionArgumentDescriptionsDefaultVersion() throws IOException {
+        // TODO: When Cypher command for getting default is available use that here
+        // for now we just force this to use the preset default
         assertResultAsJsonEquals(
                 CypherVersion.Default,
-                showFunctions(null),
+                showFunctions(CypherVersion.Default),
```

Comment:
`CypherVersion.Default` is being removed. If you really need a default one you can use:

```java
switch (GraphDatabaseSettings.default_langage) match {
  case Cypher5 => CypherVerision.Cypher5
  case Cypher25 => CypherVerision.Cypher25
}
```